Library/Formula/leo.rb

```diff
@@ -0,0 +1,26 @@
+class Leo < Formula
+  homepage "http://leoeditor.com/"
+  url "https://downloads.sourceforge.net/projects/leo/files/Leo/5.1-final/Leo-5.1-final.zip"
+  sha256 "2d742f9825959ba5c7624d1179b9f3065e14e055c90272fbce199f91770de826"
+
+  depends_on "pyqt"
+  depends_on "enchant" => :recommended
+
+  def install
+    # Obtain information on Python installation
+    python_xy = "python" + `%x(python -c "import sys;print(sys.version[:3])")`.chomp
```

Comment:
On Fri, Apr 24, 2015 at 4:25 AM, Edward K. Ream <edreamleo@gmail.com> wrote:

​> ​
Many thanks, ​Dominyk and Ludwig, for your helpful comments​.

​Here is my interpretation of your suggestions. All further suggestions
appreciated:

require 'formula'

class Leo < Formula

  homepage 'http://leoeditor.com/'
  url '
http://sourceforge.net/projects/leo/files/Leo/5.0-final/Leo-5.0-final.zip'
  sha1 '9a8ca37688e60851b4faea3adfa30aae2156eb9f'

  head https://github.com/leo-editor/leo-editor, :using => :git

  depends_on 'pyqt'
  depends_on 'enchant' => :recommended
  depends_on :python if MacOS.version <= :snow_leopard

  def install
    (lib+"python2.7/site-packages").install "leo"
    bin.install "launchLeo.py" => "leo"
  end

  test do
    system "python -c 'import leo'"
  end
end

Edward​
------------------------------------------------------------------------------------------
Edward K. Ream: edreamleo@gmail.com Leo: http://leoeditor.com/
------------------------------------------------------------------------------------------